### PR TITLE
Update kubeadm docs

### PR DESCRIPTION
**Pull request by johscheuer**
**What this PR does / why we need it**:
Some minor improvements of the kubeadm documentation

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: docs/using-controller-manager-with-kubeadm.md [hunk 1]
```diff
@@ -28,20 +30,20 @@ gzip -d openstack-cloud-controller-manager.tgz
...
-Step 4: Create a configmap with the openstack cloud configuration (see `manifests/openstack-cloud-controller-manager/cloud-config`)
+Step 4: Create a configmap with the openstack cloud configuration (see `manifests/controller-manager/cloud-config`)
 ```
 kubectl create configmap cloud-config --from-file=/etc/kubernetes/cloud-config -n kube-system
 ```
 
 Step 5: Deploy controller manager
 
-Option #1 - Using a single pod with the definition in `manifests/openstack-cloud-controller-manager/openstack-cloud-controller-manager-pod.yaml`
+Option #1 - Using a single pod with the definition in `manifests/ontroller-manager/openstack-cloud-controller-manager-pod.yaml`
```
Comment (dims) [inline docs/using-controller-manager-with-kubeadm.md]: `manifests/ontroller-manager` should be `manifests/controller-manager` (missing a 'c')

File: docs/using-controller-manager-with-kubeadm.md [hunk 2]
```diff
@@ -14,7 +14,9 @@ Then follow the usual steps to bootstrap the other nodes using `kubeadm join`
 
 Step 3: build the container image using the following (`make bootstrap` will download go SDK and glide if needed)
 ```
-make build-image
+export GOOS=linux
+make images
```
Comment (dims) [inline docs/using-controller-manager-with-kubeadm.md]: `GOOS=linux make images` should be good here. then we don't have to do the export or unset

**Comment by dims** [comment]
@johscheuer thanks for taking the time to help! there a couple of tiny nits. Will merge as soon as you update the PR.

File: docs/using-controller-manager-with-kubeadm.md (hunk 1 above)
Comment (johscheuer) [reply docs/using-controller-manager-with-kubeadm.md]: Thanks, fixed in the latest commit.

File: docs/using-controller-manager-with-kubeadm.md (hunk 2 above)
Comment (johscheuer) [reply docs/using-controller-manager-with-kubeadm.md]: Thanks, fixed in the latest commit.

**Comment by dims** [comment]
thanks!

**Pull request merged**